comma.py

```diff
@@ -0,0 +1,11 @@
+def comma(s1):
+    for f in range(len(s1)):
+        if s1[f] not in "0123456789,":
+            raise Exception("")
+        else:
+            for d in range(len(s1)):
+                if s1[d]=="," and s1[d+1]==",":
+                    raise Exception("Invalid input")
+    
+while True:
+    comma(input("Enter a string and i will validate it:"))
```

Comment:
The input message thould be clear , 1. have "i" as "I", and say explicitely that pressing "Enter" will end of the string...or similar idea"